.github/workflows/services-json.yml

```diff
@@ -0,0 +1,36 @@
+name: Services Validator
+
+on:
+  push:
+    paths-ignore:
+      - "cmake/**"
+    branches-ignore: [master]
+  pull_request:
+    paths:
+      - "plugins/rtmp-services/data/services.json"
+      - "plugins/rtmp-services/data/package.json"
+    branches-ignore: [master]
```

Comment:
The ignore filter is evaluated against the target branch, not source branch, so this is not doing what you probably intended.